vcx/wrappers/python3/demo/alice.py

```diff
@@ -27,7 +28,7 @@
 
 async def main():
 
-    payment_plugin = cdll.LoadLibrary("libnullpay.so")
+    payment_plugin = cdll.LoadLibrary("libnullpay.dylib" if platform.system() == 'Darwin' else "libnullpay.so")
```

Comment:
suggest adding const dictionary like:
`extention = {"darwin": ".dylib", "linux": ".so", "win32": ".dll"}`

Reply:
Thanks for review, I've just noticed there's already `file_ext()` to determine the postfix in demo_utils, so I've refactored it a bit and reused.